sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuOverrides.scala

```diff
@@ -1250,10 +1249,10 @@ object GpuOverrides {
       ExprChecks.unaryProjectNotLambdaInputMatchesOutput(TypeSig.STRING, TypeSig.STRING),
       (a, conf, p, r) => new UnaryExprMeta[InitCap](a, conf, p, r) {
         override def convertToGpu(child: Expression): GpuExpression = GpuInitCap(child)
-      }).incompat(CASE_MODIFICATION_INCOMPAT + " Spark also only sees the space character as " +
-      "a word deliminator, but this will capitalize any character after a non-alphabetic " + 
-      "character.  The behavior will be aligned to match Spark in the future per " +
-      "https://github.com/NVIDIA/spark-rapids/issues/2786."),
+      }).incompat("in some cases Unicode characters change byte width when changing the case." +
+      " The GPU string conversion does not support these characters for capitalize now. This" +
+      " will be fixed in the future per https://github.com/rapidsai/cudf/issues/8644. For a" +
+      " full list of unsupported characters see https://github.com/rapidsai/cudf/issues/3132."),
```

Comment:
This still refers to changing byte widths which has been fixed.  That's no longer the issue.  Now the issue is that some characters are not converted either due to a bug, like the issue you filed, or because the Unicode standard is different than the JVM.  We should drop the text about byte width changes along with pointing at issue 3132, IMO.

Reply:
Per the fix https://github.com/rapidsai/cudf/pull/8647, seems the byte widths issue still exists for `capitalize` and `title`.
I will update this after the fix being merged.

Reply:
That PR notes that it is using the same logic as upper and lower, and we can see in the code that [it is checking the number of bytes](https://github.com/rapidsai/cudf/blob/branch-21.08/cpp/src/strings/capitalize.cu#L85) for the new character.  Therefore the only issue remaining should be whether all of the characters in the conversion table match what the JVM expects (i.e.: Unicode version issues).